src/Processors/QueryPlan/ConvertingStep.h

```diff
@@ -18,6 +18,8 @@ class ConvertingStep : public ITransformingStep
 
 private:
     Block result_header;
+    /// Do not check that constants are same. Use value from result_header.
```

Comment:
This is a comment about what it does but there is no comment about why it is needed.

Reply:
Added comment.